front/app/api/custom_field_options/useCustomFieldOption.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+
+import fetcher from 'utils/cl-react-query/fetcher';
+
+import customFieldOptionsKeys from './keys';
+import { ICustomFieldOption, CustomFieldOptionsKeys } from './types';
+
+export const fetchOption = ({ optionId }: { optionId?: string }) =>
+  fetcher<ICustomFieldOption>({
+    path: `/custom_field_options/${optionId}`,
+    action: 'get',
+  });
+
+const useCustomFieldOption = ({
+  optionId,
+  enabled = true,
```

Comment:
Normally the enabled property depends on whether there is an optionId or not. I've now made the optionId optional but wanted to ask why you added the enabled property explicitly before removing it.

Reply:
This is not new code, git failed to detect the move of the files (my bad for already making some edits in the same commit). I simply left it as-is.